### ISSUE TITLE
[question / feature request] get status

### ISSUE THREAD

**Issue by panmarco83** [DavidRisch/steamvr_utils#19]
Hello again!
I have recently been wondering if it is or would be possible to get the status of the base stations (on/off) as to use a toggle method for activating and deactivating the base stations.
Do you know if this is at all possible to poll from the stations and/or if it would be something of a usable feature for the `steamvr_util.py` script ?  
  
something along the lines of: `steamvr_util.py status/active`
returning something like: ` enabled / disabled / active / inactive / true / false `  
  
What do you think ?

I recently got my hands on a stream deck and am trying to build my own version of streamdeck-ui (I just want to).
So, what I was thinking was if I had a button set to poll the status of the base stations every so and so many seconds and change the icon and/or text accordingly.

**Comment by panmarco83** [comment]
Hrm, thinking about it again, I probably should dig into your script and try to figure it out on my own, but it might be handy on your end as well ?

**Comment by DavidRisch** [comment]
> if it is or would be possible to get the status of the base stations (on/off)

As far as I know that just is not possible over Bluetooth. I looked at every value readable from a base station and none of them reliable correlated with the status. But some values changed in ways I did not understand, so just because I didn't find a way to infer the status from them does not mean its not possible.
And the firmware of my basestations is 2+ years old, so there might be new features with newer firmware.

I wish you good luck finding a solution!
